### PR TITLE
Update Kourend emoteclue to allow emberlight

### PR DESCRIPTION
**Pull request by jaybyrrd**
From Discord: 

[12:42 PM]Nightfirecat: Oh hey, if someone wants to get a PR ready for this in advance, that'd be great:
Allow players to use the Emberlight for the Treasure Trail step requiring an Arclight in the Catacombs of Kourend. 
(from the "Beyond Next Week" section of https://secure.runescape.com/m=news/while-guthix-sleeps?oldschool=1)

- Changes clause to `any`
- ~Updates text based on wiki~

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/EmoteClue.java [hunk 1]
```diff
@@ -204,7 +204,7 @@ public class EmoteClue extends ClueScroll implements LocationClueScroll
...
 		new EmoteClue("Shrug in the Shayzien war tent. Equip a blue mystic robe bottom, a rune kiteshield and any bob shirt.", "Shayzien war tent", SHAYZIEN_WAR_TENT, new WorldPoint(1487, 3635, 0), SHRUG, item(MYSTIC_ROBE_BOTTOM), item(RUNE_KITESHIELD), range("Any bob shirt", BOBS_RED_SHIRT, BOBS_PURPLE_SHIRT)),
-		new EmoteClue("Slap your head in the centre of the Kourend catacombs. Beware of double agents! Equip the arclight and the amulet of the damned.", "Kourend catacombs", CENTRE_OF_THE_CATACOMBS_OF_KOUREND, new WorldPoint(1663, 10045, 0), DOUBLE_AGENT_141, SLAP_HEAD, item(ARCLIGHT), any("Amulet of the damned", item(AMULET_OF_THE_DAMNED), item(AMULET_OF_THE_DAMNED_FULL))),
+		new EmoteClue("Slap your head in the centre of the Kourend catacombs. Beware of double agents! Equip the arclight and an amulet of the damned.", "Kourend catacombs", CENTRE_OF_THE_CATACOMBS_OF_KOUREND, new WorldPoint(1663, 10045, 0), DOUBLE_AGENT_141, SLAP_HEAD, any("Arclight or Emberlight", item(ARCLIGHT), item(EMBERLIGHT)), any("Amulet of the damned", item(AMULET_OF_THE_DAMNED), item(AMULET_OF_THE_DAMNED_FULL))),
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/EmoteClue.java:207]: Are we sure the wiki is correct about the wording? I haven't seen anyone report this clue step as broken.

Comment (Nightfirecat) [reply runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/EmoteClue.java]: Particularly given that we need to wait and be sure they permit the emberlight for that clue in their game update next week (it's likely they'll get that done in time, but not guaranteed), leaving it as a draft PR for now is ideal.

**Comment by allah09** [comment]
plz

**Review by Nightfirecat — APPROVED** [review]
LGTM; hold for game update / news post announcing this change ingame.

**Comment by Nightfirecat** [comment]
[The clue step apparently hasn't been updated yet](https://www.reddit.com/r/2007scape/comments/1e5k77x/psa_arclight_master_clue_step_still_not_working/), so we get to wait another week and see if we can bring this in next update.

**Comment by Felanbird** [comment]
The change was applied in today's update, the text of the clue now reads
```
Slap your head in the centre of the Kourend catacombs. Beware of double agents! Equip arclight or emberlight along with the amulet of the damned.
```

**Pull request closed without merging**